broker/lib/jobs/BnRStatusPollerJob.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
+
```

Comment:
Also why couldn't we refactor and reuse OperationStatusPoller itself ? Because it already has everything needed to support this use case , right ?

Reply:
OperationStatusPoller interacts with OOBManager instead of DirectorManager. Also for objectstore access the paths of metadata is different. Would be if-else in all steps. Factory pattern would lead to two more extra classes for cleaner approach. So, separated in two jobs. Also deployment lock-unlock is not present in oob deployments.